### ISSUE TITLE
Plans for integration with scalelite?

### ISSUE THREAD

**Issue by georg-e** [greenstatic/bigbluebutton-monitoring#2]
Hey,
nice work to get an overview for a single bbb instance.
Are there any plans for an integration with scalelite to support multiple bbb instances?

**Comment by greenstatic** [comment]
Currently there are no plans of supporting multiple bbb instances in the same web dashboard. 

This project was created as a quick solution to monitor a single bbb instance without any external infrastructure dependencies (e.g. Elastic Stack). At my organization we are using this in production to get a quick grasp what is going on on our two bbb instances (we currently do not have scalelite deployed). Which is what I recommend currently - having an instance deployed on each bbb instance (in our 2 bbb instance case this is feasible).

However with time we are slowly developing more 'proper' tools, such as  [bigbluebutton-exporter](https://github.com/greenstatic/bigbluebutton-exporter) which aggregates metrics using the bbb api and exports them for Prometheus. This enables us to have detailed historical metrics via Grafana for our bbb servers in combination with the hardware utilization (e.g. CPU usage vs. bbb participants). 

One shortcoming of our Prometheus exporter is however the lack of visibility into which meetings are being held, how many participants, etc. While this project does provide this information, it is only available "live" with no historical background. For this reason we are investigating logging solutions, primarily the Elastic Stack, which would allow us to have a better grasp. This includes supporting multiple bbb instances.

So in short, this project is meant for simple installations which don't require historical data. However we are investigating / looking to extend solutions to support multi bbb server monitoring using more mature logging solutions.

**Comment by greenstatic** [comment]
If however someone is keen on creating a PR to support this, I am happy to merge.

One implementation of this would be to have one monitor instance per bbb instance and then expand the frontend to be able to query multiple APIs for data. From a technical side this is not that difficult.
